### PR TITLE
feat: Lesson layout and lesson view

### PR DESCRIPTION
**Pull request by piotr-pajak**
## Jira issue(s)
[LC-500](https://selleolabs.atlassian.net/jira/software/projects/LC/boards/3?selectedIssue=LC-500)

## Preview
Screenshot 2024-12-27 at 09 33 36

[LC-500]: https://selleolabs.atlassian.net/browse/LC-500?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/api/src/seed/nice-data-seeds.ts [hunk 1]
```diff
@@ -1480,34 +1595,39 @@ export const niceCourses: NiceCourseData[] = [
...
... to analyze massive amounts of data and solve intricate problems, such as self-driving cars.</li><li><strong>Supervised vs. Unsupervised Learning:</strong><br>- <strong>Supervised Learning:</strong> The AI is trained on labeled data (e.g., images labeled as "cat" or "dog").<br>- <strong>Unsupervised Learning:</strong> The AI identifies patterns in unlabeled data without explicit instructions.</li><li><strong>Big Data:</strong> The large volume of structured and unstructured data generated by businesses and devices, which is essential for training AI models.</li><li><strong>Automation:</strong> AI is often used to automate repetitive tasks, freeing up human resources for more complex activities.</li><li><strong>Ethics in AI:</strong> As AI becomes more powerful, ensuring its ethical use (e.g., avoiding bias in decision-making) is critical for building trust.</li></ul><h3>Why These Concepts Matter</h3><p>Understanding these basic AI terms is the first step toward recognizing how AI can be applied in business. Each concept represents a building block of AI\'s potential to transform industries by increasing efficiency, improving decision-making, and creating innovative solutions.</p>',
```
Comment (typeWolffo) [inline apps/api/src/seed/nice-data-seeds.ts:1610]: as we have description as varchar(1000) this seed will fail, can make it shorter? :D

File: apps/web/app/modules/Courses/Lesson/OldLesson.page.tsx
Comment (typeWolffo) [inline apps/web/app/modules/Courses/Lesson/OldLesson.page.tsx:1]: is it restored old file during rebase?

**Review by typeWolffo — CHANGES_REQUESTED** [review]
rebase should fix that seed description length

File: apps/api/src/seed/nice-data-seeds.ts (hunk 1 above)
Comment (piotr-pajak) [reply apps/api/src/seed/nice-data-seeds.ts]: After consulting with the project owner, I increased the character limit to 3000

**Review by typeWolffo — APPROVED** [review]

**Pull request merged**